a2ml/server/README.md

```diff
@@ -0,0 +1,39 @@
+# Development
+
+## Deps
+
+Uses these packages:
+```
+asyncio
+asyncio-redis
+fastapi
+gevent
+jsonpickle
+redis
+uvicorn
+```
+
+## Test
+
+Run dev server:
+```
+cd a2ml/server
+uvicorn server:app --reload
+```
+
+Run Celery worker:
+```
+cd a2ml/tasks_queue
+celery -A celery_app worker --loglevel=info --pool=gevent -c 100
+```
```

Comment:
Same for worker, I know we don't have this right now but it would be good to have:

```
a2ml worker [options]
```